packages/newspringchurchapp/src/tabs/connect/Connect.js

```diff
@@ -34,7 +35,7 @@ class Connect extends PureComponent {
         <SafeAreaView>
           <ScrollViewWithPadding>
             <UserAvatarHeader key="UserAvatarHeaderConnected" />
-            <ActionBar />
+            {Config.EXPERIMENTAL === 'true' ? <ActionBar /> : null}
```

Comment:
Not sure why this change is in here, but I'm pretty sure we can get rid of it now since we're showing the `ActionBar` again.

Reply:
yep you're right